apps/publikator/sentry.server.config.ts

```diff
@@ -0,0 +1,15 @@
+// This file configures the initialization of Sentry on the server.
+// The config you add here will be used whenever the server handles a request.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from '@sentry/nextjs'
+
+Sentry.init({
+  dsn: 'https://064b127cfc1f7794a484500beee38359@o4505721260212224.ingest.sentry.io/4505796048453632',
```

Comment:
and this

And maybe also the existing ones in admin?

Reply:
Will look into this next week